packages/snaps-controllers/src/services/iframe/IframeExecutionService.test.browser.ts

```diff
@@ -50,9 +50,22 @@ describe('IframeExecutionService', () => {
     const iframe = document.querySelector('iframe');
     assert(iframe);
 
-    expect(iframe.contentDocument).toBeNull();
-    expect(() => iframe.contentWindow?.document).toThrow(
-      /Blocked a frame with origin ".+" from accessing a cross-origin frame./u,
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+
+    const message = new Promise((resolve) => {
+      window.addEventListener('message', (event) => {
+        resolve(event.data);
+      });
+    });
+
+    // Creates an iframe attempts to access the iframe created by the execution
+    // service. This should fail due to the sandboxing.
+    const testFrame = document.createElement('iframe');
+    testFrame.src = 'http://localhost:4567/test/sandbox';
+    document.body.appendChild(testFrame);
+
+    expect(await message).toBe(
+      'SecurityError: Blocked a frame with origin "http://localhost:4567" from accessing a cross-origin frame.',
```

Comment:
notice that the message changes among browsers, is this a problem?

Reply:
We only test in Google Chrome. Unless you think it's necessary to test in different browsers, I think this should be fine?

Reply:
that depends on snaps team goals - are we not supporting FF as well?

Reply:
We are, but the most important thing to test here is that the behaviour of our code is correct. If the test passes in Google Chrome, we know we're properly setting the sandbox property.

Reply:
I disagree. Assumptions that are made for Chrome do not by default apply to other browsers as well (especially non Chromium ones).

In other words, it is always possible that something changes in the repo and therefore sandbox setting is ruined on FF but not in Chrome and we won't know until it's too late.

Beyond just supporting FF, we also run these tests on FF, right? If not, that is the power we have when now using WDIO - Introducing FF tests support after already having Chrome tests support should be very easy.

I went through the same thing in Snow, [was very simple](https://github.com/LavaMoat/snow/pull/53/files)

I highly recommend we address that.